pom.xml

```diff
@@ -378,6 +378,24 @@ under the License.
 				<version>1.1.3</version>
 			</dependency>
 
+			<dependency>
```

Comment:
Looks good!
Only one minor comment: add a profile for this?

Reply:
very nice catch; this should of course be in a profile.